pysqldb3/pysqldb3.py

```diff
@@ -620,7 +600,7 @@ def my_tables(self, schema='public'):
             print('Aborting...attempting to run a Postgres-only command on a SQL Server/Azure DbConnect instance.')
             return
 
-        return self.dfquery(PG_MY_TABLES_QUERY.format(s=schema, u=self.user))
+        return self.dfquery(PG_MY_TABLES_QUERY.format(s=schema, u=self.user), internal = True)
```

Comment:
have you tested this? I dont think this will work, Query.dfquery returns a dataframe of Query.data, but if the query is flagged as internal, will the data will be in internal_data

Reply:
I tested it, and it was returning nothing, so I will fix that. When I made it internal, I mistakenly thought that it was for returning table names for other functions. Will test everything with some temporary tests to make sure outputs are showing up when expected